server/app/services/program/predicate/PredicateAction.java

```diff
@@ -2,6 +2,7 @@
 
 /** Action that is taken when associating predicate is evaluated to be true. */
 public enum PredicateAction {
+  ELIGIBILITY_BLOCK("eligible if"),
```

Comment:
issue (blocking): i think we need both `eligible if` and `ineligible if`

Reply:
discussed offline that ineligible isn't needed (now at least).  renamed to ELIGIBILE_BLOCK